src/adzerk/boot_cljs/middleware.clj

```diff
@@ -99,7 +99,7 @@
   (let [[path file] ((juxt core/tmppath core/tmpfile) main)
         base-name   (-> file .getName deps/strip-extension)
         js-path     (.getPath (io/file tmp-out (str base-name ".js")))
-        cljs-path   (.getPath (util/replace-path path (str base-name ".cljs")))
+        cljs-path   (.getPath (io/file (str "boot/cljs/" base-name ".cljs")))
```

Comment:
@schmir do you know if the usage of `io/file` here could cause trouble on Windows?